apps/src/sites/studio/pages/devise/registrations/_finish_sign_up.js

```diff
@@ -270,6 +275,15 @@ $(document).ready(() => {
 
   function onSchoolChange(_, event) {
     schoolData.ncesSchoolId = event ? event.value : '';
+    // ID is set to -1 and '' respectively as user toggles 'I cannot find my
```

Comment:
What does it mean here that it's set to -1 and '' respectively for the singular 'I cannot find my school above'?

Reply:
I wasn't sure how better to say this so maybe you can help! I am looking for any nces id update that isn't -1 or an empty string (hence the !includes below) because that indicates a *real school selection as opposed to a byproduct of clicking 'I cannot find my school above'.

Reply:
Oh I see gotcha! I think I read it as selecting 'I cannot find my school' can either set the ID to -1 or ''. Maybe if the comment could specify that 'I cannot find my school above' maps to -1 and setting it to the default empty maps to '' (if I'm understanding this right)?

Reply:
I'm going to update this comment in a followup. Hoping to merge this and get the new events logging before EOD:)